src/static/js/modules/util/utilities-type-checkers.js

```diff
@@ -0,0 +1,150 @@
+/* ==========================================================================
+   Javascript Type Checkers
+
+   Various utility functions to check Javascript types and primitives.
+   Copied from:
+   https://github.com/angular/angular.js/blob/master/src/Angular.js.
```

Comment:
It looks like we'll need https://github.com/angular/angular.js/blob/master/LICENSE#L3 mentioned in the top here and/or an entry in https://github.com/cfpb/cfgov-refresh/blob/flapjack/TERMS.md#exceptions